appcircle_android_resign/1.0.1/component.yaml

```diff
@@ -0,0 +1,71 @@
+platform: Android
+buildPlatform:
+displayName: Android Re-Sign
+description: "Re-sign the unsigned output file from the build step with the selected keystore to export a binary compatible with Android devices."
+docUrl: "https://docs.appcircle.io/distribute/resign/#resigning-android-binaries"
+requiredComponents:
+precedingComponents: "appcircle_android_build,appcircle_android_sign,appcircle_android_flutter_build,appcircle_android_sign"
```

Comment:
Are not these components required for resign step? If there is no generated apk/aab in the pipeline, user can not resign binary.

Reply:
@boztopuz `appcircle_android_build` or `appcircle_android_flutter_build` can be used as the **build** step, so one of them is mandatory but not both. For that I added them on `preceding`.
However, I noticed that I added the `sign` step twice, so I'll delete one of them.